server/src/main/java/org/candlepin/resteasy/filter/VerifyAuthorizationFilter.java

```diff
@@ -65,6 +65,7 @@
 @Priority(Priorities.AUTHORIZATION)
 public class VerifyAuthorizationFilter extends AbstractAuthorizationFilter {
     private static final Logger log = LoggerFactory.getLogger(VerifyAuthorizationFilter.class);
+
     private StoreFactory storeFactory;
     private ResourceLocatorMap locatorMap;
```

Comment:
locatorMap is no longer used.

Reply:
Fixed. Removed this line and the entire ResourceLocatorMap file.